src/libraries/System.ComponentModel.Annotations/ref/System.ComponentModel.Annotations.cs

```diff
@@ -217,6 +217,7 @@ public partial class RangeAttribute : System.ComponentModel.DataAnnotations.Vali
     {
         public RangeAttribute(double minimum, double maximum) { }
         public RangeAttribute(int minimum, int maximum) { }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCode("Generic TypeConverters may require the generic types to be annotated. For example, NullableConverter requires the underlying type to be DynamicallyAccessedMembers All.")]
```

Comment:
```suggestion
        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Generic TypeConverters may require the generic types to be annotated. For example, NullableConverter requires the underlying type to be DynamicallyAccessedMembers All.")]
```

repeated in the other `ref` file as well.